src/controllers/AssetInfoController.js

```diff
@@ -112,37 +119,109 @@ export class AssetInfoController {
     return assets[network][assetId];
   }
 
-  async nftInfo(address, limit = 1000) {
+  async assetFavorite(assetId) {
+    const { assets } = this.store.getState();
+    const network = this.getNetwork();
+    const asset = assets[network] && assets[network][assetId];
+
+    if (!asset) {
+      return;
+    }
+
+    assets[network][assetId].isFavorite = !asset.isFavorite;
+    this.store.updateState({ assets });
+  }
+
+  /**
+   * Force-updates storage asset info by assetId list
+   * @param {string} forAddress
+   * @param {Array<string>} assetIds
+   * @returns {Promise<void>}
+   */
+  async updateAssets(forAddress, assetIds) {
```

Comment:
`forAddress` isn't needed here, right?

Reply:
Removed it here https://github.com/wavesplatform/waves-keeper/pull/166/commits/f6b02ba7e71b8a9cf95c72c400510149ab41f084

Reply:
> `forAddress` isn't needed here, right?

Yeah, it is not necessary. Picked your commit.